src/base.py

```diff
@@ -28,5 +28,16 @@ def change_to_base(n: int, b: int) -> str:
     >>> change_to_base(31, 16)
     '1F'
     """
+    
+    final_list = []
     assert 2 <= b <= 16
-    return ''  # FIXME: return n in the right base
+    if n == 0:
+        return "0"
+    if n > 0:
+        final_list.append(n % b)
+        n = n//b
+    if n < 0:
+        negative_list.append(n % b)
+        n = n//b
+        final_list[ -i for i in negative_list ]
```

Comment:
I don't know exactly what you are trying to do here. You can't put signs on individual digits, it is not as if `-10 == -1 -0`, right? Also, this `final_list[ -i for i in ... ]` syntax is not valid Python. It looks like something you could almost do in R (although not this way), but in Python it is just a syntax error. Did you by any chance mean something like this?

```python
final_list = [ -i for i in negative_list ]
```

This is valid Python, but as I mentioned above, changing the signs on all digits doesn't make something into a negative number.